packages/common/src/settings/clipboard-history.settings.types.d.ts

```
@@ -0,0 +1,14 @@
+import {BaseSettings} from "../settings.types";
+
+export type BaseClipboardHistoryItem = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)